kafka-minion/templates/serviceaccount.yaml

```diff
@@ -9,4 +9,9 @@ metadata:
     helm.sh/chart: {{ include "kafka-minion.chart" . }}
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- if or .Values.global.imagePullSecrets .Values.serviceAccount.imagePullSecrets }}
```

Comment:
Both variables do not exist yet and therefore the deployment would fail. Can you add default variables and also add documentation for them? I think defining `.Values.global` is required or you'll run into nil pointer exceptions?